test/test-idle.c

```diff
@@ -97,3 +97,24 @@ TEST_IMPL(idle_starvation) {
   MAKE_VALGRIND_HAPPY();
   return 0;
 }
+
+
+TEST_IMPL(idle_check) {
+  ASSERT_EQ(0, uv_idle_init(uv_default_loop(), &idle_handle));
+  ASSERT_EQ(0, uv_idle_start(&idle_handle, (uv_idle_cb) uv_idle_stop));
```

Comment:
`uv_idle_stop()` and `uv_idle_cb` signatures don't match so a warning is produced.
```
In file included from test/test-idle.c:23:
test/test-idle.c: In function ‘run_test_idle_check’:
test/test-idle.c:104:44: warning: cast between incompatible function types from ‘int (*)(uv_idle_t *)’ {aka ‘int (*)(struct uv_idle_s *)’} to ‘void (*)(uv_idle_t *)’ {aka ‘void (*)(struct uv_idle_s *)’} [-Wcast-function-type]
  104 |   ASSERT_EQ(0, uv_idle_start(&idle_handle, (uv_idle_cb) uv_idle_stop));
```

Reply:
Fixed.